server/cp/ai/semaphore.py

```diff
@@ -390,38 +368,20 @@ def create_tag_in_semaphore(self, data: FeedbackData) -> ResponseType:
 
         return result_summary
 
-    @overload
-    def data_operation(  # noqa: E704
-        self,
-        verb: str,
-        operation: Literal["feedback"],
-        name: Optional[str],
-        data: FeedbackData,
-    ) -> ResponseType: ...
-
-    @overload
-    def data_operation(  # noqa: E704
-        self,
-        verb: str,
-        operation: Literal["search"],
-        name: Optional[str],
-        data: SearchData,
-    ) -> ResponseType: ...
-
     def data_operation(
         self,
         verb: str,
-        operation: Literal["search", "feedback"],
+        operation: str,
         name: Optional[str],
-        data,
+        data: OperationRequest,
     ) -> ResponseType:
         if operation == "feedback":
-            return self.create_tag_in_semaphore(data)
+            return self.analyze(data["item"])

```

Comment:
I think this will break storing of user created tags

Reply:
Resolved